applications/vessel-history/src/utils/flags.ts

```diff
@@ -0,0 +1,11 @@
+import flags from 'data/flags'
+
+type Flag = { id: string; label: string }
+export const getFlagById = (id: string): Flag | undefined => {
+  const flag = flags.find((f) => f.id === id)
```

Comment:
It would be more performant to have the flags stored in a dictionary instead of an array. As of now we are only accessing them by Id, thus it would be much faster than iterating the flags array looking for the given id. Makes sense?

Reply:
@rdgfuentes this comes from another project of the monorepo, my idea was to join this in a package then, not update it, I don't know why they implemented in this way. I know that have a list of flags (and also duplicated in two or more projects) is not the best solution, but I think the first improvement need to move this functionality (from all the apps) to a centralized place, and then see the implementation as you said, what do you think @j8seangel ?